exe/bundle_report

```diff
@@ -17,6 +17,12 @@ at_exit do
       Examples:
         #{$0} compatibility --rails-version 5.0
         #{$0} outdated
+
+      ruby_check To find a compatible ruby version for the target rails version
+
+      Examples:
+        #{$0} ruby_check --rails-version 7.0.0
```

Comment:
maybe we need to support versions that are not that specific here? I imagine using this to check for major.minor for example, not up to the patch version (like I'm in 6.0.5 and I want to check against 6.1, to not have to remember that 6.1.5 is the latest patch)

I know this is something we can get from rubygems on the file, with something like `gem list -ear rails`, it will fetch all the rails versions from the internet and we can then parse the result to get the latest matching version

Reply:
got it. will check it out.